parts/openshift/unstable/openshiftnodescript.sh

```diff
@@ -23,8 +28,3 @@ update-ca-trust
 # note: ${SERVICE_TYPE}-node crash loops until master is up
 systemctl enable ${SERVICE_TYPE}-node.service
 systemctl start ${SERVICE_TYPE}-node.service &
-
-while [[ $(KUBECONFIG=/etc/origin/node/node.kubeconfig oc get node $(hostname) -o template \
```

Comment:
removing this is problematic for the scale up case - why are you removing it?

Reply:
I'm not sure why this was removed.  Adding it back.